common/src/main/java/com/microsoft/identity/common/internal/cache/ActiveBrokerCacheUpdater.kt

```diff
@@ -71,6 +71,12 @@ class ActiveBrokerCacheUpdater(
          */
         const val BROKER_DISCOVERY_DISABLED_KEY = "broker.discovery.disabled"
 
+        /**
+         * If this key is present in the request bundle,
+         * The broker will return the active broker information in the result bundle.
+         * Note: This is only valid for requests that are processed by the BrokerOperationRequestDispatcher.
+         */
+        const val REQUEST_ACTIVE_BROKER_DATA_KEY = "com.microsoft.identity.request.broker.data"
```

Comment:
For all such keys, I think renaming them such that name starts with the word "KEY" might be better for readability. For example: `KEY_REQUEST_ACTIVE_BROKER_DATA`